addons/upgrade/to-12.1-firewallsso.pl

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/perl
+
+=head1 NAME
+
+firewallsso_to_update -
+
+=head1 DESCRIPTION
+
+firewallsso_to_update
+
+=cut
+
+use strict;
+use warnings;
+use lib qw(/usr/local/pf/lib /usr/local/pf/lib_perl/lib/perl5);
+use pf::IniFiles;
+use pf::constants::config;
+use pf::file_paths qw(
+    $firewall_sso_config_file
+);
+
+
+my $fsso = pf::IniFiles->new( -file => $firewall_sso_config_file, -allowempty => 1);
+
+if (length ($fsso->Sections()) > 0) {
+    for my $section ($fsso->Sections()) {
+        if (!($fsso->exists($section, "use_connector"))) {
+            $fsso->newval($section, 'use_connector', 'enabled');
```

Comment:
Your 'checkbox_value' is `1` so the value for `enabled` here should be `1` too

Reply:
Will do.
I just remember a code that says: enable=1=true, disable=0=false, but it may not be applied here.

Reply:
Done